src/storage/Storage-spec.ts

```diff
@@ -0,0 +1,243 @@
+import { IEvent } from '../models/IEvent';
```

Comment:
The storage tests have been updated to test both `InMemoryStorage` and `NodeFileStorage`.